src/main/java/edu/kis/powp/jobs2d/command/CommandVisitor.java

```diff
@@ -0,0 +1,27 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class CommandVisitor implements ICommandVisitor {
```

Comment:
The name should provide info about the function introduced with this particular visitor.
Executing a command is already in API o command so this visitor is redundant. Change this function to something new.